modules/localgov_media/config/optional/filter.format.wysiwyg.yml

```diff
@@ -18,58 +19,64 @@ filters:
     id: editor_file_reference
     provider: editor
     status: true
-    weight: -44
+    weight: -45
     settings: {  }
   filter_align:
     id: filter_align
     provider: filter
     status: true
-    weight: -49
+    weight: -47
     settings: {  }
   filter_autop:
     id: filter_autop
     provider: filter
     status: true
-    weight: -43
+    weight: -44
     settings: {  }
   filter_caption:
     id: filter_caption
     provider: filter
     status: false
-    weight: -42
+    weight: -41
     settings: {  }
   filter_html:
     id: filter_html
     provider: filter
     status: true
-    weight: -47
+    weight: -48
     settings:
-      allowed_html: '<br> <code> <pre> <p class="alert alert-info alert-danger alert-fail alert-success callout callout-primary callout-success callout-danger callout-teal callout-carbon callout-yellow"> <h1> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <a class="external-link pdf-link btn btn-start" hreflang title href data-entity-type data-entity-uuid data-entity-substitution> <ul class="list-checked" type> <cite> <dl> <dt> <dd> <mark> <blockquote cite> <ol type start> <img data-entity-type data-entity-uuid data-caption src alt height width data-align> <drupal-media data-caption title data-entity-type data-entity-uuid alt data-view-mode data-align> <span dir> <strong> <em> <u> <li> <table> <tr> <td rowspan colspan> <th rowspan colspan> <thead> <tbody> <tfoot> <caption>'
+      allowed_html: '<br> <p class="alert alert-info alert-danger alert-fail alert-success callout callout-primary callout-success callout-danger callout-teal callout-carbon callout-yellow"> <h1> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <a class="external-link pdf-link btn btn-start" hreflang title href data-entity-type data-entity-uuid data-entity-substitution> <ul class="list-checked" type> <cite> <dl> <dt> <dd> <mark> <blockquote cite> <ol type start> <img data-entity-type data-entity-uuid data-caption src alt height width data-align> <drupal-media data-caption title data-entity-type data-entity-uuid alt data-view-mode data-align> <span dir> <strong> <em> <u> <li> <table> <tr> <td rowspan colspan> <th rowspan colspan> <thead> <tbody> <tfoot> <caption>'
       filter_html_help: true
       filter_html_nofollow: false
   filter_html_escape:
     id: filter_html_escape
     provider: filter
     status: false
-    weight: -45
+    weight: -42
     settings: {  }
   filter_html_image_secure:
     id: filter_html_image_secure
     provider: filter
     status: false
-    weight: -40
+    weight: -39
     settings: {  }
   filter_htmlcorrector:
     id: filter_htmlcorrector
     provider: filter
     status: true
-    weight: -48
+    weight: -49
+    settings: {  }
+  filter_image_lazy_load:
```

Comment:
This is a new filter.

So we're not just reordeing the filters here.

Reply:
That lazy load is set by default when we install localgov. Should we leave it or remove it?

Reply:
This is from Drupal Core? Doesn't need enabling? Does want to be configured on here?

Then I'd say we *explicitly* add it here then.

Reply:
Can we mark this thread resolved so? And approve the PR?